k8s/alasybil.yaml

```diff
@@ -36,6 +36,8 @@ spec:
               name: secrets
               key: db
         image: docker.io/libp2p/hydra-booster
+        - name: HYDRA_DISABLE_DBCREATE
+          value: "true"
```

Comment:
These env vars are not in the right place - need to be under the `env` prop.

Reply:
doh, It was because I didn't commit my image change